temporalio/worker/activity.py

```diff
@@ -86,8 +86,9 @@ def __init__(
             # Confirm name is present
             name: Optional[str] = getattr(activity, "__temporal_activity_name", None)
             if not name:
+                fn_name = getattr(activity, "__name__", "<unknown>")
```

Comment:
NIT: I'd not take the name here and instead print the object directly, it might not even be a function.

Reply:
I mentioned in the other now-resolved thread, this gives something like `<function test_activity_without_decorator.<locals>.say_hello at 0x0000020D5FC33D90>` which is rough. That ugly string is why I didn't put the activity in the string in the first place.